Pinta/Dialogs/VersionInformationTabPage.cs

```diff
@@ -90,5 +107,55 @@ protected override void OnDestroyed ()
 
 			base.OnDestroyed ();
 		}
+
+		void CopyButton_Clicked (object sender, EventArgs e)
+		{
+			String delimeter = ",";
+			String linesep = "\n";
```

Comment:
Should this be using `Environment.NewLine` for the platform-specific line separator? (I don't actually know if the CSV spec has a requirement either way)

Reply:
done